lib/core/unittest/CTimeUtilsTest.cc

```diff
@@ -509,4 +509,155 @@ BOOST_AUTO_TEST_CASE(testDurationToString) {
                                             365 * d + 5 * h + 48 * m + 46 * s));
 }
 
+BOOST_AUTO_TEST_CASE(testTimeDurationStringToSeconds) {
+    bool parsedOk{false};
+    ml::core_t::TTime durationSeconds{0};
+    const ml::core_t::TTime defaultValue{1};
+
+    {
+        // All valid and specifying a whole number of seconds
+        std::tie(durationSeconds, parsedOk) =
+            ml::core::CTimeUtils::timeDurationStringToSeconds("14d", defaultValue);
+        BOOST_TEST_REQUIRE(parsedOk);
+        BOOST_TEST_REQUIRE(1209600, durationSeconds);
```

Comment:
There are still places where `BOOST_TEST_REQUIRE` is being used with the comma operator.

This one needs to be either `BOOST_REQUIRE_EQUAL(1209600, durationSeconds)` or `BOOST_TEST_REQUIRE(1209600 == durationSeconds)`.

But there are others too - please scan all the assertions in the PR to check they are not inadvertently using the comma operator.